src/components/userFeedback/UserFeedback.tsx

```diff
@@ -0,0 +1,63 @@
+/// <reference path="../../../types/react-tether/index.d.ts" /> Required to make dts-generator bundle react-tether definition file.
+
+import * as React from 'react';
+import { extend } from 'underscore';
+
+export interface IUserFeedbackProps {
+  feedbackText: string;
+  state: string;
+  constantClasses?: string;
+  displayOnShow?: string;
+}
+
+export interface IUserFeedbackStyle {
+  display: string;
+  className: string;
+}
+
+export const userFeedbackState = {
+  VALID: 'VALID',
+  WARNING: 'WARNING',
+  ERROR: 'ERROR',
+};
+
+export class UserFeedback extends React.Component<IUserFeedbackProps, any> {
+
+  private getDefaultUserFeedbackStyle(): IUserFeedbackStyle {
+    return {
+      display: 'none',
+      className: this.props.constantClasses ? this.props.constantClasses + ' text-dark-grey' : 'text-dark-grey'
```

Comment:
Since `text-dark-grey` is used both in the if and the else, I would write it: `'text-dark-grey ' + (this.props.constantClasses ? this.props.constantClasses : '')`